yt_dlp/extractor/tubitv.py

```diff
@@ -119,6 +119,15 @@ def _real_extract(self, url):
             'release_year': int_or_none(video_data.get('year')),
         }
 
+        episode_match = re.search(r'^S(\d+):E(\d+) - (.+)', title)
+
+        if episode_match:
+            infodict['season_number'] = int(episode_match.group(1))
+            infodict['episode_number'] = int(episode_match.group(2))
+            infodict['episode_title'] = episode_match.group(3)
+
+        return infodict
```

Comment:
```py
        season_number, episode_number, episode_title = self._search_regex(
            r'^S(\d+):E(\d+) - (.+)', ..., groups=(1, 2, 3), default=(None, None, None))
        return {
            ...
            'episode': episode_title,
            ...
        }
```

Reply:
Didn't know about this syntax. Very cool, added it